manualdirectdebit.php

```diff
@@ -0,0 +1,152 @@
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_postInstall
+ */
+function manualdirectdebit_civicrm_postInstall() {
+  _manualdirectdebit_civix_civicrm_postInstall();
+}
+
+/**
+ * Implements hook_civicrm_uninstall().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_uninstall
+ */
+function manualdirectdebit_civicrm_uninstall() {
+  _manualdirectdebit_civix_civicrm_uninstall();
+}
+
+/**
+ * Implements hook_civicrm_enable().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_enable
+ */
+function manualdirectdebit_civicrm_enable() {
+  _manualdirectdebit_civix_civicrm_enable();
+}
+
+/**
+ * Implements hook_civicrm_disable().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_disable
+ */
+function manualdirectdebit_civicrm_disable() {
+  _manualdirectdebit_civix_civicrm_disable();
+}
+
+/**
+ * Implements hook_civicrm_upgrade().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_upgrade
+ */
+function manualdirectdebit_civicrm_upgrade($op, CRM_Queue_Queue $queue = NULL) {
+  return _manualdirectdebit_civix_civicrm_upgrade($op, $queue);
+}
+
+/**
+ * Implements hook_civicrm_managed().
+ *
+ * Generate a list of entities to create/deactivate/delete when this module
+ * is installed, disabled, uninstalled.
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_managed
+ */
+function manualdirectdebit_civicrm_managed(&$entities) {
+  _manualdirectdebit_civix_civicrm_managed($entities);
+}
+
+/**
+ * Implements hook_civicrm_caseTypes().
+ *
+ * Generate a list of case-types.
+ *
+ * Note: This hook only runs in CiviCRM 4.4+.
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_caseTypes
+ */
+function manualdirectdebit_civicrm_caseTypes(&$caseTypes) {
+  _manualdirectdebit_civix_civicrm_caseTypes($caseTypes);
+}
+
+/**
+ * Implements hook_civicrm_angularModules().
+ *
+ * Generate a list of Angular modules.
+ *
+ * Note: This hook only runs in CiviCRM 4.5+. It may
+ * use features only available in v4.6+.
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_angularModules
+ */
+function manualdirectdebit_civicrm_angularModules(&$angularModules) {
+  _manualdirectdebit_civix_civicrm_angularModules($angularModules);
+}
+
+/**
+ * Implements hook_civicrm_alterSettingsFolders().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_alterSettingsFolders
+ */
+function manualdirectdebit_civicrm_alterSettingsFolders(&$metaDataFolders = NULL) {
+  _manualdirectdebit_civix_civicrm_alterSettingsFolders($metaDataFolders);
+}
+
+// --- Functions below this ship commented out. Uncomment as required. ---
+
+/**
+ * Implements hook_civicrm_preProcess().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_preProcess
+ *
+function manualdirectdebit_civicrm_preProcess($formName, &$form) {
```

Comment:
please remove these commented out functions (manualdirectdebit_civicrm_preProcess and manualdirectdebit_civicrm_navigationMenu)

Reply:
ok